src/openforms/js/utils/fetch.js

```diff
@@ -113,6 +113,11 @@ const put = async (url, csrftoken, data = {}, throwOn400 = false) => {
   return resp;
 };
 
+const patch = async (url, csrftoken, data = {}, throwOn400 = false) => {
```

Comment:
new function -> let's go for the optimal interface from the start!

```suggestion
const patch = async (url, csrftoken, data = {}, throwOn400 = true) => {
```